zio-http/src/main/scala/zhttp/http/HttpData.scala

```diff
@@ -80,14 +80,14 @@ object HttpData {
   /**
    * Helper to create HttpData from String
    */
-  def fromString(text: String, charset: Charset = HTTP_CHARSET): HttpData = Text(text, charset)
+  def fromString(text: CharSequence, charset: Charset = HTTP_CHARSET): HttpData = Text(text, charset)
```

Comment:
Keep this, and let's add a `fromCharSequence` also.

Reply:
Ok great